src/options/style.css

```diff
@@ -1,10 +1,95 @@
-body{
-    font-family: var(--default-font);
+/* Import Google Fonts - Inter */
+@font-face {
+    font-family: 'Inter';
+    src: url('../assets/inter/Inter-Regular.woff2') format('woff2'),
+        url('../assets/inter/Inter-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
+/* Import Font Awesome */
+@font-face {
+    font-family: 'Font Awesome 6 Free Solid';
```

Comment:
This is not working on my side, the icon only comes when I keep the link code in the HTML